steps/assets/openstack/resources/rm-assets.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+bootstrap_cleanup() {
+    rm /etc/systemd/system/multi-user.target.wants/rm-assets.service
+    rm /etc/systemd/system/tectonic.service
+    rm /etc/systemd/system/bootkube.service
+    rm /etc/systemd/system/rm-assets.service
+    rm -rf /opt/tectonic
+}
```

Comment:
Since none of this looks OpenStack specific, it would be nice to use a common cleanup script at some point

Reply:
:+1: absolutely